lib/identity/client.js

```diff
@@ -172,6 +172,42 @@ class Client extends PartialClient {
     let resultQuery = await this.storage.search(request)
     return resultQuery
   }
+
+  /**
+   * getSharedSecrets
+   *
+   *
+   * @return {Promise<Record>}
+   */
+  async getSharedSecrets() {
```

Comment:
Does this need it's own method or can we think about adding filters to the list method?

Reply:
agree with where your mind is heading, @lkwdwrd @rosasck we need to chat about this before we merge this code in